gpt_index/indices/vector_store/pinecone.py

```diff
@@ -99,12 +99,6 @@ def _add_document_to_index(
 
         id_node_embed_tups = self._get_node_embedding_tups(nodes, set())
         for new_id, node, text_embedding in id_node_embed_tups:
-            # assign a new_id if current_id conflicts with existing ids
```

Comment:
@jerryjliu I fork this codebase just to remove these lines. 

Why are these here?

Reply:
(said in a polite tone)

Reply:
Thank you also for all of this work!